index.html

```diff
@@ -7,14 +7,69 @@
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     </head>
     <body>
         <main>
             <nav id="navbar">
                 <img src="images/RickAndMortyLogo.png" alt="Rick and Morty logo PNG" id="logo">
+                <ul id="navbarList">
+                    <li id="allCharBtn">All Characters</li>
+                </ul>
+                <div id="search-field">
+                    <div id="search-bar">
+                        <input type="text" id="search-input" placeholder="Search by name">
+                        <button id="search-btn" class="material-icons">
+                            <span class="material-icons">search</span>
+                        </button>
+                    </div>
+                    <button class="filter-btn" id="show-filter-btn">Show filters<i class="arrow down" id="filter-arrow"></i></button>
+                    <div class="filters-tab">
+                        <div id="status-div">
+                            <h3 class="filter">Status:</h3>
+                            <select name="status-select" id="status-select">
+                                <option value=""></option>
+                                <option value="alive">Alive</option>
```

Comment:
Idealmente, la api deberia tener un endpoint que devuelve las opciones que tienen los filtros (no se si este lo tiene, creo que no). Si lo tuviera, usas ese endpoint para mostrar las opciones y asi, si el dia de mañana agregan una opcion nueva, no tenes que hacer cambio vos en el front end, se hace solo en el BE. 
Cuando los valores estan asi estaticos se dice que estan "hardcoded" o "hardcodeados" en latinoamerica. Generalmente esta mal visto

Reply:
No estoy seguro en el caso de HTML/Javascript, pero creo que si vas a hardcodear los valores, es mejor hacerlo en el archivo JS (es decir, tener una lista en JS y de ahi cargar las opciones en el HTML). Eso despues te da mas flexibilidad si necesitas, por ejemplo, traducir los campos

Reply:
Por lo que entendí de la documentación, no hay un endpoint que devuelve los filtros que existen, solo te explican cuales hay y como accederlos.